Metrics Note for ECS (#8649)
DataDog/documentation: content/en/agent/amazon_ecs/data_collected.md

### Metrics

> Amazon ECS metrics can be collected by the Agent or when using the [AWS integration][1].

Reviewer: ```suggestion
Amazon ECS metrics are collected through the Datadog Agent or [AWS integration][1].
```

> In the table below, metrics collected when using the AWS integration are prefixed with `aws.*`. Metrics collected by the Agent are prefixed with `ecs.*`.

Reviewer: ```suggestion
Metrics collected using the AWS integration are prefixed with `aws.*`. Metrics collected by the Agent are prefixed with `ecs.*`. See the table below:
```
